src/main/resources/application-base.yml

```diff
@@ -196,6 +196,8 @@ postponed:
     interval: 3600000
     maxAttempts: 10
 authenticator:
+  user:
+    registrationStrategy: CREATE_IF_FIRST
```

Comment:
I would add this into README into the auth section. I believe, these things are very related